vignettes/ChromBand.Rmd

```diff
@@ -0,0 +1,437 @@
+---
+title: "Using Chromosome Bands as Categories"
+author: 
+   - name: D. Sarkar
+   - name: S. Falcon
+   - name: R. Gentleman
+   - name: Kemmy MO Jones
+     affiliation: [ Chromosome Bands Vignette translation from Sweave to R Markdown / HTML ]
+date: "`r format(Sys.Date(), '%B %d, %Y')`"
+package: Category
+vignette: >
+  %\VignetteIndexEntry{Using Chromosome Bands as Categories}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+output:
+  BiocStyle::html_document:
+    toc: false
+    theme: journal
```

Comment:
Remove line 18. We use the default theme so that all vignettes use the same styling.

Reply:
Ok